pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1424,7 +1407,6 @@
       "AssemblyVersionInPackageVersion": {
         "4.0.0.0": "4.0.0",
         "4.0.10.0": "4.0.10",
-        "4.0.11.0": "4.0.11",
         "4.0.12.0": "4.0.12"
```

Comment:
This was inconsistent with the repo state.  We never shipped this version stable and we've moved past it.